docs/en/stack/ml/anomaly-detection/ml-troubleshooting.asciidoc

```diff
@@ -188,3 +188,39 @@ POST _ml/set_upgrade_mode?enabled=false&timeout=10m
 DELETE temp_ml_config
 --------------------------------------------------
 // TEST[skip:TBD]
+
+[discrete]
+[[ml-debian8-memory]]
+== Suboptimal job assignment on Debian 8
+
+Where possible {ml} jobs are assigned to nodes based on the memory requirement
```

Comment:
I read this as "...possible ML jobs...", which didn't make sense, so I suggest adding a comma here:
```suggestion
Where possible, {ml} jobs are assigned to nodes based on the memory requirement
```